arches/management/commands/packages.py

```diff
@@ -1114,6 +1124,128 @@ def import_reference_data(self, data_source, overwrite="ignore", stage="stage",
+                        setattr(instance, field_name, valuetype)
+                    elif field_name == "language":
+                        related_language = models.Language.objects.get(code=value)
+                        setattr(instance, field_name, related_language)
+                    else:
+                        related_list_item = models.ControlledListItem.objects.get(id=value)
+                        setattr(instance, field_name, related_list_item)
+                else:
+                    setattr(instance, field_name, value)
+
+            # run validation on all non-parent fields & gather for bulk create
+            instance.clean_fields(exclude={"parent"})
+            instances.append(instance)
+        
+        model.objects.bulk_create(instances)
+
+        if model == models.ControlledListItem:
+            # Create list item relationships after all list items have been created
+            for child, parent in list_items_with_parent.items():
+                child.parent = models.ControlledListItem.objects.get(id=parent)
+                child.clean_fields(exclude={field["name"] for field in fields if field["name"] != "parent"})
+                child.save()
+        
+    
+    def export_controlled_lists(self, data_dest):
+        wb = openpyxl.Workbook()
+        ws = wb.active
+        ws.title = "ControlledList"
+        self.export_model_to_sheet(ws, models.ControlledList)
+        self.export_model_to_sheet(wb, models.ControlledListItem)
+        self.export_model_to_sheet(wb, models.ControlledListItemValue)
+        
+        # if data_dest == ".":
+        #     data_dest = os.path.dirname(settings.SYSTEM_SETTINGS_LOCAL_PATH)
+        if data_dest != "" and data_dest != ".":
+            wb.save(os.path.join(data_dest, "controlled_lists.xlsx"))
+            self.stdout.write("Data exported successfully to controlled_lists.xlsx")
+        else:
+            utils.print_message("No destination directory specified. Please rerun this command with the '-d' parameter populated.")
+            sys.exit()
+
+    def export_model_to_sheet(self, wb, model):
+        # For the first sheet (ControlledList), use blank sheet that is initiallized with workbook
+        # otherwise, append a new sheet
+        if isinstance(wb, openpyxl.worksheet.worksheet.Worksheet):
+            ws = wb
+        else:
+            ws = wb.create_sheet(title=model.__name__)
+        fields = [{"name": field.name, "datatype": field.get_internal_type()} for field in model._meta.fields]
+        ws.append(field["name"] for field in fields)
+        for instance in model.objects.all():
+            row_data = []
+            for field in fields:
+                value = getattr(instance, field["name"])
+                if (isinstance(value, models.ControlledList) or isinstance(value, models.ControlledListItem) or isinstance(value, models.ControlledListItemValue)):
```

Comment:
You can pass an iterable of types to isinstance
```suggestion
                if isinstance(value, (models.ControlledList, models.ControlledListItem, models.ControlledListItemValue)):
```